src/dashboard/Dashboard.tsx

```diff
@@ -91,6 +105,7 @@ const Dashboard = ({
                     <NeoDashboardTitle />
                     <NeoDashboardHeaderPageList />
                     <NeoPage></NeoPage>
+                    <Chat></Chat>
```

Comment:
lets to create this using the extension interface

Reply:
i'm taking this out for now, we'll handle this in another round